### PR TITLE
Instagram command

### PR DESCRIPTION
**Pull request by metehus**
image
image

### CODE REVIEW

File: src/apis/Instagram.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  getUser (user) {
+    return this.request(user)
+  }
+
+  request (endpoint, queryParams = {}) {
+    const qParams = new URLSearchParams({
+      ...queryParams,
+      __a: 1
+    })
+    return fetch(API_URL + endpoint + `?${qParams.toString()}`)
```
Comment (almeidx) [inline src/apis/Instagram.js:22]: axios?

File: src/utils/CanvasTemplates.js [hunk 2]
```diff
@@ -3,13 +3,14 @@ const Color = require('./Color.js')
...
 const moment = require('moment')
+const { loadImage } = require('canvas')
 
 let Canvas = {}
 let CanvasUtils = {}
 try {
   Canvas = require('canvas')
   CanvasUtils = require('./CanvasUtils.js')
-} catch (e) {}
+} catch (e) { }
```
Comment (almeidx) [inline src/utils/CanvasTemplates.js:13]: 👀

**Comment by almeidx** [comment]
Btw, resolve the conflicts

File: src/apis/Instagram.js (hunk 1 above)
Comment (metehus) [reply src/apis/Instagram.js:22]: why 😭

Comment (almeidx) [reply src/apis/Instagram.js:22]: - Axios performs automatic transformations of JSON data.
- Axios is more secure, featuring built in cross site forgery (XSRF) protection
- Axios has better error handling
- Axios has interceptors
- Axios has the ability to monitor POST request progress

File: src/utils/CanvasTemplates.js [hunk 3]
```diff
@@ -913,4 +914,34 @@ module.exports = class CanvasTemplates {
...
+    ctx.fillRect(0, 0, SIZE, SIZE)
+
+    const images = await Promise.all(urls.map(i => loadImage(i)))
+
+    const GAPS_SIZE = GAP * (TILES - 1)
+    const TILE_SIZE = ((SIZE - GAPS_SIZE) / TILES)
+
+    let count = 0
+    for (let i = 0; i < TILES; i++) {
+      for (let j = 0; j < TILES; j++) {
```
Comment (almeidx) [inline src/utils/CanvasTemplates.js:935]: nested loops 😨

**Review by almeidx — APPROVED** [review]

File: src/apis/Instagram.js (hunk 1 above)
Comment (metehus) [reply src/apis/Instagram.js:22]: - said @fernando457829

File: src/utils/CanvasTemplates.js (hunk 3 above)
Comment (metehus) [reply src/utils/CanvasTemplates.js:935]: what do you want me to fucking do?

**Review by Doges — APPROVED** [review]
LGTM

**Pull request merged**